### PR TITLE
moving src and adding codegen

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- apply source and config to new layout
- adding codegen for channel/source

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: contrib/test/e2e/helpers/kafka_helper.go [hunk 1]
```diff
@@ -36,8 +36,8 @@ import (
 	testlib "knative.dev/eventing/test/lib"
 	pkgtest "knative.dev/pkg/test"
 
-	sourcesv1beta1 "knative.dev/eventing-kafka/contrib/kafka/source/pkg/apis/sources/v1beta1"
-	kafkaclientset "knative.dev/eventing-kafka/contrib/kafka/source/pkg/client/clientset/versioned"
+	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go [hunk 2]
```diff
@@ -36,8 +36,8 @@ import (
...
+	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
```
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go [hunk 3]
```diff
@@ -23,11 +23,11 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	bindingsv1alpha1 "knative.dev/eventing-kafka/contrib/kafka/source/pkg/apis/bindings/v1alpha1"
-	bindingsv1beta1 "knative.dev/eventing-kafka/contrib/kafka/source/pkg/apis/bindings/v1beta1"
-	sourcesv1alpha1 "knative.dev/eventing-kafka/contrib/kafka/source/pkg/apis/sources/v1alpha1"
-	sourcesv1beta1 "knative.dev/eventing-kafka/contrib/kafka/source/pkg/apis/sources/v1beta1"
-	kafkaclientset "knative.dev/eventing-kafka/contrib/kafka/source/pkg/client/clientset/versioned"
+	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go [hunk 4]
```diff
@@ -23,11 +23,11 @@ import (
...
+	bindingsv1alpha1 "knative.dev/eventing-kafka/pkg/apis/bindings/v1alpha1"
+	bindingsv1beta1 "knative.dev/eventing-kafka/pkg/apis/bindings/v1beta1"
+	sourcesv1alpha1 "knative.dev/eventing-kafka/pkg/apis/sources/v1alpha1"
+	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
```
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: pkg/channel/consolidated/reconciler/controller/resources/service.go [hunk 5]
```diff
@@ -22,8 +22,9 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/eventing-kafka/pkg/apis/messaging/v1beta1"
-	"knative.dev/eventing/pkg/utils"
 	"knative.dev/pkg/kmeta"
+	"knative.dev/pkg/network"
+
```
Comment (mattmoor) [inline pkg/channel/consolidated/reconciler/controller/resources/service.go:27]: Format Go code:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 1 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:39]: Format Go code:
```suggestion

```

File: contrib/test/e2e/helpers/kafka_helper.go (hunk 2 above)
Comment (mattmoor) [inline contrib/test/e2e/helpers/kafka_helper.go:40]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: contrib/test/lib/creation.go (hunk 3 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:26]: Format Go code:
```suggestion

```

File: contrib/test/lib/creation.go (hunk 4 above)
Comment (mattmoor) [inline contrib/test/lib/creation.go:30]: Format Go code:
```suggestion
	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
```

File: pkg/channel/consolidated/reconciler/controller/resources/service.go (hunk 5 above)
Comment (mattmoor) [inline pkg/channel/consolidated/reconciler/controller/resources/service.go:27]: Format Go code:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

**Comment by matzew** [comment]
Tested END-TO-END as well

**Comment by slinkydeveloper** [comment]
go fmt is out of place, can you do a followup to fix it @matzew ?

/lgtm

**Pull request merged**